GNUmakefile

```diff
@@ -163,12 +163,18 @@ all.arm64: apiserver-image.arm64 exporter-image.arm64 pool-mgmt-image.arm64 volu
            admission-server-image.arm64 cspc-operator-image.arm64 upgrade-image.arm64 \
            cvc-operator-image.arm64 cspi-mgmt-image.arm64 provisioner-localpv-image.arm64
 
+
 .PHONY: initialize
 initialize: bootstrap
 
 .PHONY: deps
 deps:
-	dep ensure
+	@echo "--> Syncing vendor directory"
+	@go mod vendor
+	@echo "--> Tidying up submodules"
+	@go mod tidy
+	@echo "--> Veryfying submodules"
+	@go mod verify
```

Comment:
need to change the sequence here. 
- go mod tidy
- go mod verify
- go mod vendor